src/AbstractWebApplication.php

```diff
@@ -878,6 +878,11 @@ protected function detectRequestUri()
 	 */
 	protected function header($string, $replace = true, $code = null)
 	{
+		if (is_null($code))
```

Comment:
```suggestion
		if ($code !== null)
```
We normally don't use `is_null()`.